docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -399,6 +401,53 @@ For example, if you work with Keycloak, you can use `keycloak.js` to authenticat
 </html>
 ----
 
+==== Use OIDC proxy
+
+Instead of tying your SPA to a specific OIDC provider by this OIDC provider's script, you may want to consider implementing an authorization code flow directly in the SPA by redirecting the users to a configurable OIDC authorization endpoint, accepting the OIDC redirect in the SPA and completing the authorization code flow by sending the code to a standard OIDC token endpoint and finaly sending the acquired access tokens to a Quarkus endpoint.
```

Comment:
Yeah, this sentence is too long (and confusing too), Vale is on point for once :)

Reply:
@FroMage Sure, I've tried to make it simpler and clearer across the whole SPA section